src/toM3u8.js

```diff
@@ -3,6 +3,9 @@ import { findIndexes } from './utils/list';
 import { addSegmentsToPlaylist } from './segment/segmentBase';
 import { byteRangeToString } from './segment/urlType';
 
+export const generateSidxKey = (sidx) => sidx &&
```

Comment:
We need this function in VHS, we re-implemented it there which was fairly pointless imo.